script/prerender/contentPages.tsx

```diff
@@ -203,9 +196,31 @@ function injectHTML(html: string, {body, styles, state, fonts, meta, modules, ti
 
   const assetManifest = JSON.parse(readAssetFile('asset-manifest.json'));
 
-  const extractAssets = () => Object.keys(assetManifest)
-    .filter((asset) => modules.indexOf(asset.replace('.js', '')) > -1)
-    .map((k) => assetManifest[k]);
+  /*
+   * separate chunks are automatically made for vendor code
+   * (https://facebook.github.io/create-react-app/docs/production-build)
+   *
+   * Loadable.preloadReady() only waits for the react-loadable chunks,
+   * apparently some of those are triggering the load of some of the
+   * numbered chunks and pre-rendering breaks as it waits for them
+   * to download.
+   *
+   * i'm not aware of any way to tell which ones we need ahead of time
+   * and render script tags for them here, so we're just loading all
+   * the numbered chunks.
+   *
+   * it would probably be better to wait for completion of any chunks
+   * that are requested dynamically, but i'm not seeing a way in
+   * webpack's api to do that.
+   */
+  const extractAssets = () => Object.keys(assetManifest.files)
+    .filter((asset) =>
+      // chunks requested by react-loadable
+      modules.indexOf(asset.replace('.js', '')) > -1
```

Comment:
any reason not to use `includes` rather than the old `indexOf != -1` ?

Reply:
only that i forgot `includes` is a thing